client/my-sites/stats/post-performance/index.jsx

```diff
@@ -192,3 +182,15 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state, ownProps ) => {
+	const { site } = ownProps;
+	const query = { status: 'published' };
```

Comment:
Curious about the reasoning for passing in `query` as a prop here.  I believe this component will always operate for published posts only.